### PR TITLE
Update deps, tooling, code style

### PR DESCRIPTION
**Pull request by briancavalier**
Update allthethings!
- eslint config
- rollup + buba + buble
- npm scripts
- code style
- test consolidation to appease jsinspect

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,13 +9,14 @@
     "type-defintions/dom-event.d.ts"
   ],
   "scripts": {
-    "build-dist": "mkdir -p dist && rollup src/dom-event.js | babel --presets es2015 --plugins transform-es2015-modules-umd --module-id '@most/dom-event' -o dist/dom-event.js",
-    "build": "npm run build-dist && uglifyjs dist/dom-event.js -o dist/dom-event.min.js",
+    "build": "npm run build-dist && uglifyjs dist/mostPackage.js -o dist/mostPackage.min.js",
+    "build-dist": "mkdir -p dist && rollup -c --name '@most/mostPackage' | umd-name-transform -o dist/mostPackage.js",
```
Comment (TylorS) [inline package.json]: mostPackage -> mostDomEvent

File: package.json [hunk 2]
```diff
@@ -9,13 +9,14 @@
     "type-defintions/dom-event.d.ts"
   ],
   "scripts": {
-    "build-dist": "mkdir -p dist && rollup src/dom-event.js | babel --presets es2015 --plugins transform-es2015-modules-umd --module-id '@most/dom-event' -o dist/dom-event.js",
-    "build": "npm run build-dist && uglifyjs dist/dom-event.js -o dist/dom-event.min.js",
+    "build": "npm run build-dist && uglifyjs dist/mostPackage.js -o dist/mostPackage.min.js",
```
Comment (TylorS) [inline package.json]: mostPackage -> mostDomEvent

File: package.json (hunk 1 above)
Comment (TylorS) [inline package.json]: `rollup -c` assumes that there is a file `rollup.config.js` is it not neeed here?

Comment (briancavalier) [inline package.json]: oops!  forgot to copy it from package-starter.  Thanks!

File: package.json [hunk 3]
```diff
@@ -51,7 +53,8 @@
     "most": "^1.0.1",
     "rollup": "0.34.8",
     "rollup-plugin-buble": "0.12.1",
-    "uglify-js": "^2.7.1"
+    "uglify-js": "^2.7.1",
+    "umd-name-transform": "1.0.1"
```
Comment (TylorS) [inline package.json]: You can remove umd-name-transform

**Pull request merged**